runhouse/resources/hardware/launcher_utils.py

```diff
@@ -135,6 +135,10 @@ def run_verbose(
                 spinner.stop()
                 spinner = None
 
+            if event.event == "log_spinner":
+                spinner = Spinner(logger=logger, desc=str(event.data))
```

Comment:
Would you want to include a line above it to persist in the logs after the spinner stops (similar to `info_spinner`):
```
cls.log_processor.log_event(event, cluster_name=cluster_name)
```

Reply:
actually not sure if we do, bc the log itself is meaningless in that situation (for event type of log spinner i mean). or am i missing something?

Reply:
Nvm, I think it makes sense without that based on the other PRs!